HelloWorld.java

```diff
@@ -36,5 +36,6 @@
 public class HelloWorld {
   public static void main(String[] args) {
     System.out.println("Hello, World");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://docker.home.zuerker.org:9000/sonar/coding_rules#rule_key=squid%3AS106)